modules/ossm-about-concepts-resources.adoc

```diff
@@ -78,6 +76,35 @@ You can think of the relationship between the `Istio` and `IstioRevision` resour
 
 Similarly, users create an `Istio` resource which instructs the {SMProduct} Operator to create a matching `IstioRevision` resource, which then in turn triggers the creation of the Istio control plane. To do that, the {SMProduct} Operator will copy all of your relevant configuration from the `Istio` resource to the `IstioRevision` resource.
 
+[id="istiorevisiontag-resource_{context}"]
+== IstioRevisionTag resource
+
+The `IstioRevisionTag` resource represents a stable revision tag, that functions as an alias for Istio control plane revisions. With the stable tag, `prod`, you can use the label `istio.io/rev=prod` to inject proxies into your workloads. When you perform an upgrade to a control plane with a new revision name, you can update your tag to point to the new revision instead of having to relabel your workloads and namespaces. For more information, see "Stable Revision Tags."
```

Comment:
```suggestion
The `IstioRevisionTag` resource represents a stable revision tag that functions as an alias for Istio control plane revisions. With the stable tag, `prod`, you can use the label `istio.io/rev=prod` to inject proxies into your workloads. When you perform an upgrade to a control plane with a new revision name, you can update your tag to point to the new revision instead of having to relabel your workloads and namespaces. For more information, see "Stable Revision Tags."
```